cmd/kubeadm/app/util/pkiutil/pki_helpers_test.go

```diff
@@ -804,3 +904,97 @@ func TestRemoveDuplicateAltNames(t *testing.T) {
 		}
 	}
 }
+
+func TestVerifyCertChain(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir")
+	}
+	defer os.RemoveAll(tmpdir)
+
+	rootCert1, rootKey1, err := NewCertificateAuthority(&CertConfig{
+		Config: certutil.Config{CommonName: "Root CA 1"},
+	})
+	if err != nil {
+		t.Errorf("failed to create root CA cert and key with an error: %v", err)
```

Comment:
t.Fatalf

Reply:
Good catch, I copied that from some of the other tests. I've updated it for my tests and other tests where I think it should be `Fatalf`: https://github.com/kubernetes/kubernetes/pull/97266/commits/625491fe5ab296185448937464427ae1c3532840